### PR TITLE
ASTT-85 Code clean up

### PR DESCRIPTION
**Pull request by AbednigoLethole**
This pull request covers work to delete files that are not used by the ASTT and their references. It also includes bug fix that was taking place when pointing (For the AZ/EL graphs could not display values when pointing) .

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,8 @@ FROM ubuntu:20.04
 ENV DEBIAN_FRONTEND=noninteractive
 
 # Apt update
-RUN apt-get update -y
+RUN apt-get update -y &&  apt-get install --fix-missing
```
Comment (USilere) [inline Dockerfile:8]: This flags reinstalls missing packages? in what case would we have a missing file or package, maybe a bad configuration from our dependencies?

File: src/astt_gui/app.py [hunk 2]
```diff
@@ -76,14 +76,14 @@ def start_astt_gui():
             logger.error("Error encountered : %s", err)
 
         global THREAD
-        start_thread(THREAD, cm.antenna_mode)
+        start_thread(THREAD, cm.antenna_node)
```
Comment (USilere) [inline src/astt_gui/app.py:79]: Thanks for the catch, it was my negligence.

File: src/astt_gui/app.py [hunk 3]
```diff
@@ -162,6 +162,7 @@ def get_current_datetime():
 
 def background_thread(node):
     """Feed az & el to the GUI."""
+    print(node)
```
Comment (USilere) [inline src/astt_gui/app.py:165]: Not necessary, you can remove.

**Review by USilere — APPROVED** [review]
Tested the changes, they look good on my end.

File: Dockerfile (hunk 1 above)
Comment (AbednigoLethole) [reply Dockerfile:8]: No, it looks for packages in different IP address so it can pull all of them

**Review by USilere — APPROVED** [review]
Thanks for these changes.

**Pull request merged**